<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>421</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="pictureLabel_2">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>170</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>170</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pictureLabel_1">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>170</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>170</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonStop">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonEnd">
      <property name="text">
       <string>结束</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
